cloudferrylib/os/network/neutron.py

```diff
@@ -386,7 +404,7 @@ def convert_subnets(snet, cloud):
         res_hash = network_res.get_resource_hash(result,
                                                  'name',
                                                  'enable_dhcp',
-                                                 'allocation_pools',
```

Comment:
Our hash function have worked wrong and always hashed `allocation_pools` **word** :smile: 

After fixing this bug in current pull request, we can not hash this value, because it is `dict` type (that is not hashable). But we still have `cidr` value hashed.